test/system/255-auto-update.bats

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests for automatically update images for containerized services
+#
+
+load helpers
+
+UNIT_DIR="/usr/lib/systemd/system"
+
+function setup() {
+    skip_if_remote "systemd tests are meaningless over remote"
+    skip_if_rootless
+
+    basic_setup
+}
+
+function generate_service() {
+    fake_img=$1
+    target_img=$2
+    cmd=$3
+    label=$4
+
+    cname=$(random_string)
+    echo $cname
+    run_podman tag $fake_img $target_img
+    if [ "$4" ]; then
+        run_podman run -d --name $cname --label "io.containers.autoupdate=$label" $target_img $cmd
```

Comment:
...and, even better: instead of this duplication, do something like
```bash
if [[ -n $label ]]; then
    label="--label io.containers.autoupdate=$label"
fi
run_podman run -d --name $cname $label $target_img $cmd
```

Reply:
OK. Thanks for your comments. Will update this.